getting_started/step_by_step/signals.rst

```diff
@@ -393,12 +393,12 @@ Here is the complete ``sprite_2d.gd`` file for reference.
             Position += velocity * (float)delta;
         }
 
-        private void OnButtonPressed()
+        private void _on_button_pressed()
         {
             SetProcess(!IsProcessing());
         }
 
-        private void OnTimerTimeout()
+        private void _on_timer_timeout()
```

Comment:
This absolutely won't work as this is connected via script above, did you actually test this code?

Reply:
changing the delegate and inserting method into the delegate did work for timer either way so I stand corrected on the timer. I do apologize I on that bit so the original did work on this.

The OnButtonPressed does seem to be bugged. I will look into whether I have a version issue, or some issue in the way my project is handled and comment little later.